ci-operator/step-registry/aro-hcp/provision/delete-dev-env/aro-hcp-provision-delete-dev-env-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Create an ARO HCP Cluster + Node pool using bicep.
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set -x # Turn on command tracing
+
+# read the secrets and login as the user
+export TEST_USER_CLIENT_ID; TEST_USER_CLIENT_ID=$(cat /var/run/hcp-integration-credentials/client-id)
+export TEST_USER_CLIENT_SECRET; TEST_USER_CLIENT_SECRET=$(cat /var/run/hcp-integration-credentials/client-secret)
+export TEST_USER_TENANT_ID; TEST_USER_TENANT_ID=$(cat /var/run/hcp-integration-credentials/tenant)
+az login --service-principal -u "${TEST_USER_CLIENT_ID}" -p "${TEST_USER_CLIENT_SECRET}" --tenant "${TEST_USER_TENANT_ID}"
+
+make infra.svc.clean || true
+make infra.mgmt.clean || true
+make infra.region.clean || true
+
```

Comment:
There is a `SHARED_DIR` env var that we can place files into in the create step and use in the delete step to clean up. We can also add other steps to gather debugging.